src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,32 +1,66 @@
 package org.usfirst.frc4904.robot;
 
-import org.usfirst.frc4904.standard.custom.controllers.CustomXbox;
 
+import org.usfirst.frc4904.robot.subsystems.FloorIO;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
+import org.usfirst.frc4904.standard.custom.PCMPort;
+import org.usfirst.frc4904.standard.custom.controllers.CustomJoystick;
+import org.usfirst.frc4904.standard.custom.controllers.CustomXbox;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CANTalonSRX;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 
 public class RobotMap {
-    public static class Port {
-        public static class HumanInput {
-            public static final int xboxController = 1;
-
-        }
-        public static class CANMotor {}
-        public static class PWM {}
-        public static class CAN {}
-        public static class Pneumatics {}
-    }
-    
-    public static class Metrics{}
-    public static class Component {
-        public static CustomXbox driverXbox;
-    }
-    public static class HumanInput {
-        public static class Driver {
-            public static CustomXbox xbox;
-        }
-        public static class Operator {}
-    }
-    public RobotMap() {
-        Component.driverXbox = new CustomXbox(Port.HumanInput.xboxController);
-		Component.driverXbox.setDeadZone(0.1);
-    }
+	public static class Port {
+		public static class HumanInput {
+			public static final int xboxController = 1;
+		}
+
+		public static class CANMotor {
+			public static final int hatchRoller = 14; // TODO: Adjust port numbers
+			public static final int cargoRoller = 15; // TODO: Adjust port numbers
+		}
+
+		public static class PWM {
+		}
+
+		public static class CAN {
+		}
+
+		public static class Pneumatics {
+			public static final PCMPort velcroPiston = new PCMPort(0, 1, 0); // TODO: Adjust port numbers
+			public static final PCMPort pushPiston = new PCMPort(0, 3, 2); // TODO: Adjust port numbers
+			public static final PCMPort wrist = new PCMPort(-1, -1, -1); // TODO: Adjust port numbers
+		}
+	}
+
+	public static class Metrics {
+	}
+
+	public static class Component {
+		public static FloorIO floorio;
+		public static Motor cargoRollerIntake;
+		public static Motor hatcherRollerIntake;
```

Comment:
You never set these fields, and they're publicly accessible through floorio, so this declaration should be deleted (of cargo and hatch rollers)